schema/pnts.featureTable.schema.json

```diff
@@ -40,6 +43,9 @@
             },
             "CONSTANT_RGBA" : {
                 "$ref" : "featureTable.schema.json#/definitions/globalPropertyCartesian3"
+            },
+            "BATCH_LENGTH" : {
```

Comment:
Is this property necessary? Can't we just assume that the batchIds provided are correct?